packages/api/src/webhooks/cannon.ts

```diff
@@ -167,7 +185,7 @@ export default class WebhookCannon {
       await this._fireHook(trigger, false);
     } catch (err) {
       console.log("_fireHook error", err);
-      await this.retry(trigger);
+      await this.retry(trigger, err);
```

Comment:
I think you're sending `err` for a `webhookPayload` param. Shouldn't this be?
```suggestion
      await this.retry(trigger, null, err);
```

This should also be detected by TypeScript automatically when you set the right types to the inputs of the `retry` function. I'll make another suggestion there.